Controllers/ProductController.js

```diff
@@ -0,0 +1,34 @@
+const DbService = require("../Services/DbService");
+const ProductService = require("../Services/ProductsService");
+const JsonResponseService = require("../Services/JsonResponseService");
+const DataCheckers = require("../Services/SanitiseValidateService");
+
+let getAllProducts = async(req, res) => {
+    let connection = await DbService()
```

Comment:
Again 'const' for those items of information that should be set.

control + cmd + G enables the selection of all of the same things to be changed in a single instance